File: nodes/array-length/array-length.process.py
async def process(inputs, settings, config, nodeConfig):
    """
    Process function for the Array Length node.
    """
    array = inputs.get("array", [])

    # Default length to 0 if not an array
    length = 0

    # Check if the input is an array and calculate its length
    if isinstance(array, list):
        length = len(array)

    return {"length": length}

File: sdks/nodejs/nodes/array-length/array-length.process.js
export default async ({inputs, settings, config}) => {

  const array = inputs.array;

  // Initialize the length
  let length = 0;

  // Check if the input is an array and calculate its length
  if (Array.isArray(array)) {
    length = array.length;
  }

  return { length };
};
